backend/globaleaks/models/serializers.py

```diff
@@ -385,3 +388,68 @@ def serialize_tenant(session, tenant, config=None):
         ret.update(ConfigFactory(session, tenant.id).serialize('tenant'))
 
     return ret
+
+
+def serialize_auditlog(log):
+    return {
+        'date': log.date,
+        'type': log.type,
+        'user_id': log.user_id,
+        'object_id': log.object_id,
+        'data': log.data
+    }
+
+
+def serialize_comment_log(log):
+    """
+    Serialize an audit log entry for external use.
+    """
+    return {
+        'id': log['object_id'],
+        'creation_date': log['date'],
+        'content': '',
+        'data': log.get('content', 'changed'),
+        'author_id': log["user_id"],
+        'visibility': 'public',
+        'type': 'auditlog',
+        'log_type': log['type']
+    }
+
+
+def get_label(session, label_id, table):
```

Comment:
This function is not being used anywhere. Consider removing it and adding when needed.